src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25193.cs

```diff
@@ -22,5 +21,4 @@ public void BackgroundGradientsShouldRenderCorrectly()
 		App.WaitForElement("label");
```

Comment:
This test is failing on macOS:
`Snapshot different than baseline: BackgroundGradientsShouldRenderCorrectly.png (size differs - baseline is 789x563 pixels, actual is 1920x1051 pixels)`